RiotSwiftUI/Modules/UserSessions/UserSessionsOverview/View/UserSessionsOverview.swift

```diff
@@ -56,7 +56,26 @@ struct UserSessionsOverview: View {
         }
     }
     
-    var otherSessionsSection: some View {
+    private func currentSessionsSection(currentSessionViewData: UserSessionCardViewData) -> some View {
```

Comment:
This is entirely a pet peeve of mine so totally ignorable, but I’d favour making this into an `@ViewBuilder var` and putting the `if let` inside that so it doesn’t need to use a function to make a view.